programs/server/Server.cpp

```diff
@@ -575,6 +575,7 @@ static void sanityChecks(Server & server)
 int Server::main(const std::vector<std::string> & /*args*/)
 try
 {
+    setThreadName("ClickHouse");
```

Comment:
Did you want to set the main thread name to "Main"? (as in description to the PR)

Reply:
Ah, did read all your comments.